projects/telemetry/src/main.c

```diff
@@ -3,40 +3,23 @@
 #include <stdint.h>
 #include "can_hw.h"
 #include "can_uart.h"
-#include "delay.h"
 #include "event_queue.h"
-#include "gpio.h"
 #include "gps.h"
 #include "interrupt.h"
 #include "log.h"
 #include "nmea.h"
 #include "soft_timer.h"
 #include "uart.h"
-#include "wait.h"
-
-GpioSettings telemetry_settings_gpio_general = {
-  .direction = GPIO_DIR_OUT,  // The pin needs to output.
-  .state = GPIO_STATE_LOW,    // Start in the "off" state.
-  .alt_function = GPIO_ALTFN_1,
-};
 
 UartSettings telemetry_gps_uart_settings = {
   .baudrate = 9600,
-  .tx = { .port = GPIO_PORT_A, .pin = 2 },
-  .rx = { .port = GPIO_PORT_A, .pin = 3 },
-  .alt_fn = GPIO_ALTFN_1,
-};
-
-// The pin numbers to use for providing power and turning the GPS on and off
-GpioAddress telemetry_gps_pins[] = {
-  { .port = GPIO_PORT_B, .pin = 3 },  // Pin GPS power
-  { .port = GPIO_PORT_B, .pin = 4 },  // Pin GPS on_off
+  .tx = { .port = GPIO_PORT_B, .pin = 10 },
+  .rx = { .port = GPIO_PORT_B, .pin = 11 },
+  .alt_fn = GPIO_ALTFN_4,
```

Comment:
Can you add a comment clarifying the alternate function setting? i.e. what GPIO_ALTFN_4 entails